src/test/java/stepDefs/BookFragmentDefinitions.java

```diff
@@ -1,21 +1,24 @@
 package stepDefs;
 
+import desktop.fragments.BookElement;
 import desktop.fragments.BookTableFragment;
 import io.cucumber.java.en.Then;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class BookFragmentDefinitions {
 
-    @Then("verify author name - {string} in block - {string} with book - {string} and click add button")
-    public void verifyAuthorAndAddToBasket(String name, String block, String book){
-        BookTableFragment bookTableFragment = new BookTableFragment(block, book);
-        String nameAuthor = bookTableFragment.getAuthorText();
+    @Then("verify that book - {string} in block - {string} have author - {string} and add to basket")
+    public void verifyAuthorAndAddToBasket(String nameBook, String block, String author){
+        BookTableFragment bookTableFragment = new BookTableFragment(block);
```

Comment:
Try to get used to using common layers.
Defenotions -> Actions -> Pages -> Fragments and so on